pmd-ruleset/src/main/resources/pmd-ruleset/ruleset.xml

```diff
@@ -12,13 +12,19 @@
     xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
   <description>PMD ruleset originally created for insight-brain</description>
 
-  <rule ref="rulesets/java/unusedcode.xml">
-    <!-- This rule appears to be flaky and tends to give false positives involving static methods -->
-    <exclude name="UnusedPrivateMethod"/>
-  </rule>
-  <rule ref="rulesets/java/imports.xml">
-    <exclude name="TooManyStaticImports"/>
-    <exclude name="UnnecessaryFullyQualifiedName"/>
-  </rule>
-  <rule ref="rulesets/java/braces.xml"/>
+  <!-- Unused code -->
+  <rule ref="category/java/bestpractices.xml/UnusedImports" />
+  <rule ref="category/java/bestpractices.xml/UnusedFormalParameter" />
+  <rule ref="category/java/bestpractices.xml/UnusedLocalVariable" />
+  <rule ref="category/java/bestpractices.xml/UnusedPrivateField" />
```

Comment:
Comparing the [ruleset documentation for the old version of pmd](https://pmd.github.io/pmd-5.4.6/pmd-java/rules/index.html#Unused_Code) to this, it looks like UnusedModifier is missing.

Reply:
> Comparing the ruleset documentation for the old version of pmd to this, it looks like UnusedModifier is missing.

We used to use maven-pmd-plugin 3.8, which uses PMD 5.6.1. The `unusedcode` ruleset in 5.6.1 contains only four rules:
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/index.html#Unused_Code